packages/button/src/Button.spec.js

```diff
@@ -144,25 +144,25 @@ describe('Button', () => {
 
     describe('.icon', () => {
         it('should add `has-icon` if an icon is provided', () => {
-            const button = mount(<Button icon="bell" />);
+            const button = mount(<Button icon="alarm-bell" />);
```

Comment:
dit zijn geen icons uit de ai-file? of heb je ze niet allemaal geprefixed met ai-?

Reply:
Het prefix `ai-` wordt automatisch toegevoegd als het niet aanwezig is ;-)

Reply:
Is dat niet "gevaarlijk"/"vervelend" als je je eigen icoon klasse wilt zetten?

Reply:
Misschien wel. Morgen even bespreken?